dask_sql/input_utils/hive.py

```diff
@@ -126,7 +126,7 @@ def wrapped_read_function(location, column_information, **kwargs):
             else:  # pragma: no cover
                 # prevent python to optimize it away and make coverage not respect the
                 # pragma
-                dummy = 0
+                dummy = 0  # noqa: F841
```

Comment:
I understand that this is here for sake of coverage, but is there any other way we can retain the `else` block without create an unused variable?